MdePkg/Library/FltUsedLib/Readme.md

```diff
@@ -0,0 +1,15 @@
+# FltUsedLib
+
+This library provides a global (fltused) that needs to be defined anywhere floating point operations are used.
+The C compiler produces the _fltused symbol by default, this is just to satisfy the linker.
```

Comment:
Can you please move this info to the PR description. The info there now is not that helpful/relevant.

Reply:
Might want to point to this file in the integration instructions there too.

Reply:
Moved the information to the PR description.  Also pointed to the new Readme file.